shared/nav.ios.js

```diff
@@ -43,16 +43,20 @@ function NavigationBarRouteMapper (navigateTo, navigateUp) {
         return null
       }
 
-      const previousRoute = navState.routeStack[index - 1]
-
       return (
-        <NativeTouchableOpacity
-          onPress={() => route.upLink ? navigateTo(route.upLink) : navigateUp()}
-          style={styles.navBarLeftButton}>
-          <Text type='Body' style={{...styles.navBarText, ...styles.navBarButtonText}}>
-            {route.upTitle || route.leftButtonTitle || previousRoute.title || 'Back'}
-          </Text>
-        </NativeTouchableOpacity>
+        <ClickableBox
+          onClick={() => route.upLink ? navigateTo(route.upLink) : navigateUp()}
+          style={styles.navBarLeftButton}
+          underlayColor={globalColors.white}>
+          <Box>
```

Comment:
cleaned this up since i'm touching it. makes it match our design more closesly. you default to getting the <- Back